files/fr/web/css/_colon_-moz-locale-dir(rtl)/index.md

```diff
@@ -13,7 +13,7 @@ translation_of: Web/CSS/:-moz-locale-dir(rtl)
 
 {{Non-standard_header}}{{CSSRef}}
 
-La [pseudo-classe CSS](/fr/docs/Web/CSS/Pseudo-classes) **`:-moz-locale-dir(rtl)`** correspond à un élément si l'interface utilisateur est affichée de droite à gauche. Elle est déterminée par la préférence `intl.uidirection.`_`locale`_ (où _`locale`_ est la langue courante) qui est fixée à `"rtl"`.
+La [pseudo-classe CSS](/fr/docs/Web/CSS/Pseudo-classes) `:-moz-locale-dir(rtl)` correspond à un élément si l'interface utilisateur est affichée de droite à gauche. Elle est déterminée par la préférence `intl.uidirection.locale` (où `locale` est la langue courante) qui est fixée à `"rtl"`.
```

Comment:
It looks like we're keeping emphasis around these, cf. [the en-US version](https://github.com/mdn/content/pull/19885/files#diff-5aa2ba284091185b9b54382a21eebcf0dbdf1b76215096aae0753b6f06e1c158R19)

```suggestion
La [pseudo-classe CSS](/fr/docs/Web/CSS/Pseudo-classes) **`:-moz-locale-dir(rtl)`** correspond à un élément si l'interface utilisateur est affichée de droite à gauche. Elle est déterminée par la préférence `intl.uidirection.locale` (où `locale` est la langue courante) qui est fixée à `"rtl"`.
```